src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -282,6 +282,27 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
     executeOrResume(backendCall, runIdForResumption = runId)
   }
 
+  def useCachedCall(cachedCall: BackendCall, backendCall: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] = Future {
+    val log = workflowLoggerWithCall(backendCall)
+    authenticateAsUser(backendCall.workflowDescriptor) { interface =>
+      Try(interface.copy(cachedCall.callGcsPath, backendCall.callGcsPath)) match {
+        case Failure(ex) =>
+          log.error(s"Exception occurred while attempting to copy outputs from ${cachedCall.callGcsPath} to ${backendCall.callGcsPath}", ex)
+          FailedExecutionHandle(ex)
+        case Success(_) => postProcess(backendCall) match {
+          case Success(outputs) => SuccessfulExecutionHandle(outputs, backendCall.downloadRcFile.get.stripLineEnd.toInt, backendCall.hash)
+          case Failure(ex: AggregatedException[_]) if ex.exceptions.map(_.exception).isInstanceOf[SocketTimeoutException] =>
```

Comment:
I don't think this can ever be a `SocketTimeoutException`, `ex.exceptions.map(_.exception)` is going to be a `Seq[Throwable]`, you probably want `ex.exceptions.map(_.exception).exists(_.isInstanceOf[SocketTimeoutException])`